ironmon_tracker/constants/MoveData.lua

```diff
@@ -4996,7 +4996,7 @@ MoveData.MOVES_MASTER_LIST = {
 		pp = "10",
 		accuracy = "100",
 		category = MoveData.MOVE_CATEGORIES.SPECIAL,
-		description = "Inflicts regular damage. Has 10% chance to poison the target."
+		description = "Inflicts regular damage. Hits all adjacent pokemon. Has 10% chance to poison each target."
```

Comment:
I think this could sound a bit smoother if the first 2 sentences were compounded, e.g. "Inflicts regular damage and hits all adjacent Pok\233mon.", or "Inflicts regular damage, hitting all adjacent Pok\233mon."

Reply:
Went with the first option